docs/source/api/spatial-functions.rst

```diff
@@ -1789,3 +1832,152 @@ st_zmin
     :type col: Column
     :rtype: Column: DoubleType
 
+    :example:
+
+.. tabs::
+    .. code-tab:: py
+
+     >>> df = spark.createDataFrame([{'wkt': 'POINT (30 10 20)'}])
+     >>> df.select(st_zmin('wkt')).show()
+     +-----------------+
+     |st_minmaxxyz(wkt)|
```

Comment:
@milos-colic wouldn't you just want this to be `st_zmin(wkt)` for the result column name?

Reply:
for now we cant do this, it was unnecessarily abstracted, the base class is to intertwined.
I will migrate this to independent expression in a separate PR.